### DISCUSSION TITLE
Latency improvement with Ipex

### DISCUSSION THREAD

**Discussion by kiko0217** [mindee/doctr#942] [Ideas]
in other case, Doctr use in server without GPU so must improve speed. maybe. can Doctr be approved with Ipex?, or make version lite.
https://github.com/intel/intel-extension-for-pytorch

**Comment by felixdittrich92** [comment]
Hi @kiko0217, 
we have on track for 0.6.0 to make all models ONNX exportable and providing a way to use this exported models directly in docTR 👍 (than your "lite" models will be the quantized versions)
